certification/code/certificates.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+'''
+Generate a certificate from a template.
+
+* Requires the python package 'cairosvg' to be installed.
+  Please visit http://cairosvg.org/ for install instructions.
+* Some systems may also need to have 'cairo' installed.
+  Please visit http://cairographics.org/download/ for the same.
+'''
+
+import sys
+import os
+import re
+import tempfile
+import cairosvg
+from django.conf import settings
+
+#---------------------------------------------------------------
```

Comment:
This line contains one of PEP8 violations. This is not a big deal, but we try to follow PEP8 rules in AMY.

You can probably install PEP8 plugin to your editor of choice to help you validate Python syntax against PEP8.